system-contracts/scripts/compile-yul.ts

```
@@ -1,30 +1,21 @@
 // hardhat import should be the first import in the file
 import type { CompilerPaths } from "./utils";
-import {
-  spawn,
-  compilerLocation,
-  prepareCompilerPaths,
-  getSolcLocation,
-  needsRecompilation,
-  setCompilationTime,
-} from "./utils";
+import { spawn, compilerLocation, prepareCompilerPaths, needsRecompilation, setCompilationTime } from "./utils";
 import * as fs from "fs";
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)